src/main/java/gndata/lib/util/change/Merge.java

```diff
@@ -0,0 +1,65 @@
+package gndata.lib.util.change;
+
+import java.util.*;
+
+import com.hp.hpl.jena.ontology.OntModel;
+import com.hp.hpl.jena.rdf.model.*;
+import com.hp.hpl.jena.vocabulary.*;
+
+/**
+ * Class that knows how to update Metadata entities.
+ */
+public class Merge implements Change {
+
+    private Model new_object;
+    private OntModel ontology;
+    private boolean applied = false;
+
+    private Model changes = ModelFactory.createDefaultModel();
+
+    public Merge(Model new_object, OntModel ontology) {
+        this.new_object = new_object;
+        this.ontology = ontology;
+    }
+
+    public void applyTo(Model m) {
+        // ensure there is only one triple that defines type
+        List<Resource> all = new_object.listSubjectsWithProperty(RDF.type).toList();
+        assert(all.size() == 1);
+        Resource subj = all.get(0);  // this should be the actual Subject
+
+        // ensure all triples are about the same subj
+        Set<Resource> st = new_object.listSubjects().toSet();
+        st.remove(subj);
+        assert st.isEmpty();
```

Comment:
Uhhhh assert! Please read this carefully:
http://docs.oracle.com/javase/8/docs/technotes/guides/language/assert.html#usage